sixtools/info_functions.py

```diff
@@ -0,0 +1,106 @@
+from blueksy import simulators
+
+
+def list_scans(plan, db):
+    '''Print a summary of a plan, with some SIX specific info
+
+    Prints a version of the plan, showing moves that occur outside of runs, and
+    potential scan id's for any runs. Note that the scan id's listed assume
+    that the first scan has the next scan_id inthe sequence, if there is an
+    aborted scan, or if other scans occur first, this may not be true.
+
+    If a 'reason' kwarg is included in the metadata this is also included in
+    the printout.
+
+    For example the expected printout for each run should be:
+    "scan no XXXX, 'reason', motor1 = Y1, motor2 = Y2, ......."
+        where XXXX is the expected scan ID, 'reason' is an optional string
+        given by the 'reason' kwarg in he scans metadata and Yi is the value
+        that motor1 was set too prior to running the scan.
+
+    A partial reference to this should be made in the startup folder using:
+
+    .. codeblock:: python
+       from sixtools import list_scans
+       def list_scans = partial(list_scans, db=db)
+
+
+    Parameters
+    ----------
+    plan: iterable
+        must yield'Msg' objects
+
+    db: databroker.
+        The databroker that the scans are being logged in.
+
+    '''
+
+    in_run = 0  # A way to track if we are in a run or not.
```

Comment:
Would

`in_run = False
`
Make the code more readable?

Reply:
This is true, so I have made the change.